src/components/Choice/Choice.tsx

```diff
@@ -35,20 +35,14 @@ export default function Choice({
   helpText,
   onClick,
 }: Props) {
-  function handleClick() {
```

Comment:
I feel like this logic is out of scope for the `Choice` component, so it was moved to the `handleInput ` function in `Checkbox`